docs/Bread.md

```diff
@@ -0,0 +1,89 @@
+## What is BREAD?
+
+BREAD is the community currency for the Breadchain ecosystem which exists on Gnosis Chain. All BREAD is created through the [Bread Crowdstaking Application](https://www.notion.so/Crowdstaking-Application-9f233bc2fb1e419ebeb58a2809b21658?pvs=21) which anyone with xDAI on Gnosis Chain is able to use to have some BREAD for themselves.
```

Comment:
So whenever I see a fully capitalized name like this, I assume it is referring to the symbol/ticker. I think we should either prepend the $ to any occurrence for consistency personally.

Bread ($BREAD) is the community currency for the Breadchain ecosystem, deployed to the [Gnosis Chain](https://www.gnosischain.com/) network as an [ERC20 token](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/). All $BREAD is created through the [Bread Crowdstaking Application](https://www.notion.so/Crowdstaking-Application-9f233bc2fb1e419ebeb58a2809b21658?pvs=21). Anyone with [xDAI](https://docs.gnosischain.com/about/tokens/xdai/) on Gnosis Chain can bake $BREAD for themselves, which is referred to as "baking".